disperser/cmd/batcher/main.go

```diff
@@ -169,18 +170,23 @@ func RunBatcher(ctx *cli.Context) error {
 	}
 	finalizer := batcher.NewFinalizer(config.TimeoutConfig.ChainReadTimeout, config.BatcherConfig.FinalizerInterval, queue, client, rpcClient, config.BatcherConfig.MaxNumRetriesPerBlob, 1000, config.BatcherConfig.FinalizerPoolSize, logger, metrics.FinalizerMetrics)
 	var wallet walletsdk.Wallet
-	if len(config.FireblocksConfig.APIKey) > 0 &&
-		len(config.FireblocksConfig.SecretKeyPath) > 0 &&
+	if len(config.FireblocksConfig.APIKeyName) > 0 &&
+		len(config.FireblocksConfig.SecretKeyName) > 0 &&
 		len(config.FireblocksConfig.BaseURL) > 0 &&
 		len(config.FireblocksConfig.VaultAccountName) > 0 &&
-		len(config.FireblocksConfig.WalletAddress) > 0 {
-		secretKey, err := os.ReadFile(config.FireblocksConfig.SecretKeyPath)
+		len(config.FireblocksConfig.WalletAddress) > 0 &&
+		len(config.FireblocksConfig.Region) > 0 {
```

Comment:
In terms of fallback to EOA wallet, looks like we would need to *delete* one of these FireblocksConfig secrets - which seems not ideal. A cleaner alternative might be to add check on env var like `FIREBLOCKS_DISABLED=1` that can be easily updated & deployed via helm chart to trigger fallback to EOA wallet.